src/Illuminate/Support/Str.php

```diff
@@ -1211,6 +1211,17 @@ public static function ucsplit($string)
         return preg_split('/(?=\p{Lu})/u', $string, -1, PREG_SPLIT_NO_EMPTY);
     }
 
+    /**
+     * Makes the first character of each word in a string uppercase.
+     *
+     * @param  string  $string
+     * @return string
+     */
+    public static function ucwords($string)
+    {
+        return collect(explode(' ', $string))->map(fn ($word) => static::ucfirst($word))->join(' ');
```

Comment:
Using native functions `explode/array_map/implode` directly might be slightly more performant without loosing much readability.